Fix unit test in CI with fallback token.

Found actual issue for #9 and #10 

```
java.io.IOException: canceled due to java.lang.IllegalStateException: Please provide access token in `local.properties`.
	at ???(Coroutine boundary.?(?)
	at dev.hossain.githubstats.io.ClientTest$given empty timeline response - provides empty timeline events$1.invokeSuspend(ClientTest.kt:84)
	at kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt$runTestCoroutine$2.invokeSuspend(TestBuilders.kt:212)
Caused by: java.io.IOException: canceled due to java.lang.IllegalStateException: Please provide access token in `local.properties`.
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:530)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: java.lang.IllegalStateException: Please provide access token in `local.properties`.
		at dev.hossain.githubstats.io.Client.getAccessToken(Client.kt:90)
		at dev.hossain.githubstats.io.Client.access$getAccessToken(Client.kt:24)
		at dev.hossain.githubstats.io.Client$okHttpClient$$inlined$-addInterceptor$1.intercept(OkHttpClient.kt:1085)
		at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
		at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201)
		at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:517)
		... 3 more
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -6,6 +6,9 @@ on:
   pull_request:
     branches: [ "main" ]
 
+env:
+  IS_GITHUB_CI: true
+
 jobs:
   build:
 
@@ -37,4 +40,4 @@ jobs:
       run: ./gradlew lintKotlin
 
     - name: Build with Gradle
-      run: ./gradlew assemble
+      run: ./gradlew build

--- a/.idea/gradle.xml
+++ b/.idea/gradle.xml
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>

--- a/src/main/kotlin/dev/hossain/githubstats/io/Client.kt
+++ b/src/main/kotlin/dev/hossain/githubstats/io/Client.kt
@@ -81,12 +81,11 @@ object Client {
             val properties = Properties()
             properties.load(propertiesFile.inputStream())
 
-            if (properties.containsKey("access_token").not()) {
-                throw IllegalStateException("Please provide access token in `local.properties`.")
-            }
-
-            return properties.getProperty("access_token")
+            return properties.getProperty("access_token", "MISSING-TOKEN")
         } else {
+            if (System.getenv("IS_GITHUB_CI") == "true") {
+                return "CI-JOB-ON-GITHUB-ACTION"
+            }
             throw IllegalStateException("Please provide access token in `local.properties`.")
         }
     }

